flocker/node/_docker.py

```diff
@@ -480,6 +485,23 @@ def exists(self, unit_name):
         container_name = self._to_container_name(unit_name)
         return deferToThread(self._blocking_exists, container_name)
 
+    def _blocking_container_runs(self, container_name):
```

Comment:
Probably not highest priority concern.  As long as the test suite runs the logging code without unexpected exceptions it's probably okay for the time being.